news_flash/.idea/dictionaries/user.xml

```diff
@@ -0,0 +1,16 @@
+<component name="ProjectDictionaryState">
```

Comment:
those files in `.idea` belong to intellij and shouldn't be in git.
put `.idea/` in a new `.gitignore` file inside `news_flash` (or make the intellij project use the parent dir as the root)

Reply:
Fixed